File: app/src/main/java/com/gmail/fattazzo/formula1world/fragments/BaseFragment.java
package com.gmail.fattazzo.formula1world.fragments;

import android.support.v4.app.Fragment;

import com.gmail.fattazzo.formula1world.fragments.home.HomeFragment_;
import com.gmail.fattazzo.formula1world.utils.FragmentUtils;

/**
 * @author fattazzo
 *         <p/>
 *         date: 19/10/17
 */
public class BaseFragment extends Fragment {

    public void backPressed() {
        FragmentUtils.replace(getActivity(), new HomeFragment_());
    }
}

File: app/src/main/kotlin/com/gmail/fattazzo/formula1world/fragments/BaseFragment.kt
package com.gmail.fattazzo.formula1world.fragments

import android.support.v4.app.Fragment

import com.gmail.fattazzo.formula1world.fragments.home.HomeFragment_
import com.gmail.fattazzo.formula1world.utils.FragmentUtils

/**
 * @author fattazzo
 *
 *
 * date: 19/10/17
 */
open class BaseFragment : Fragment() {

    open fun backPressed() {
        FragmentUtils.replace(activity, HomeFragment_())
    }
}
